components/ActionModals/ActionModal.vue

```diff
@@ -71,17 +71,24 @@
             />
           </FormGroup>
         </form>
-        <div v-else-if="session.sessionType === SESSION_TYPES.KEPLR">
-          The transaction will be send to the Keplr Browser Extension for
-          signing.
-        </div>
-        <div v-else-if="session.sessionType === SESSION_TYPES.EXTENSION">
-          The transaction will be send to the Lunie Browser Extension for
-          signing.
-        </div>
-        <div v-else-if="session.sessionType === SESSION_TYPES.LEDGER">
-          The transaction will be sent to the Ledger Nano for signing.
-        </div>
+        <Card v-else-if="session.sessionType === SESSION_TYPES.KEPLR">
+          <div slot="subtitle">
+            The transaction will be sent to the Keplr Browser Extension for you
```

Comment:
why not just switch the content and reduce code?

Reply:
a function with a switch inside. Much better, yes

Reply:
good suggestion!